joplin/base/wagtail_hooks.py

```diff
@@ -19,7 +19,11 @@ def editor_css():
 
 @hooks.register('insert_editor_js')
 def editor_js():
-    return format_html('<script src="{}">', static('js/admin.js'))
+    urls = [
+        'https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.4/lodash.min.js',
```

Comment:
what are we using lodash for in the admin js? Should we plan to eventually pull in a copy locally?

Reply:
I haven't made the PR yet, but I'm gonna use it for live preview. This addition accidentally got caught up in this commit.

I'm fairly indifferent in this case about where lodash is served from. I feel like we'll eventually have a legit build step for JS in wagtail because things will get complex enough to warrant it. Thoughts? I'm happy to add a local copy.